Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -185,10 +194,23 @@ props.buildReportOrder.each { buildReportFile ->
 					rootDir = output[0].trim()
 					file = output[1].trim()
 					deployType = output[2].trim()
+					String file = buildRecord.getFile()
+					String owningApplication = file.split('/')[0];
```

Comment:
owningApplication should be `props.application`. It is an existing parameter that is passed to the script.